### ISSUE TITLE
WebView in QML

### ISSUE THREAD

**Issue by White-Oak** [filcuc/DOtherSide#47]
Original issue: https://github.com/White-Oak/qml-rust/issues/9

As far as I can see, to use WebView in QML one should call `QtWebView::initialize();` after creating `QApplication`.

So the goal is to provide a function to call that one. The problem is -- dependencies. AFAIK, DOtherSide would require QtWebView in system to provide this function, but probably not everyone would have it (embedded devices are an example). Is it possible to hide that feature behind some compilation flag? Say, `make DOtherSide --qtwebview`.

**Comment by filcuc** [comment]
Yes, but i would prefear an extra dll. Like DOtherSideExtra..For supporting
bindings of Qt stuff not strictly related to the pure Qml stuff.

Il 26 set 2016 13:35, "Oak" <notifications@github.com> ha scritto:

**Comment by White-Oak** [comment]
@filcuc sounds good.

**Comment by filcuc** [comment]
This should be possible after #52
